awx/main/models/execution_environments.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from awx.api.versioning import reverse
+from awx.main.models.base import PrimordialModel
+
+
+__all__ = ['ExecutionEnvironment']
+
+
+class ExecutionEnvironment(PrimordialModel):
+    class Meta:
+        unique_together = ('organization', 'image')
+        ordering = (models.F('organization_id').asc(nulls_first=True), 'image')
+
+    organization = models.ForeignKey(
+        'Organization',
+        null=True,
+        default=None,
+        blank=True,
+        on_delete=models.CASCADE,
+        related_name='%(class)ss',
+        help_text=_('The organization used to determine access to this execution environment.'),
+    )
+    image = models.CharField(
+        max_length=1024,
+        verbose_name=_('image location'),
```

Comment:
I would suggest, "Image name, which may include registry and tag"

Reply:
Generally the form field label gets set to this value, so do we really want to use something this verbose?

Reply:
I would really like to see some content in the `docs/` folder with the exact syntax that is expected to work.

Take https://quay.io/repository/ansible-tower/ansible-runner as an example, we're sandboxing with this in other places too. Would there be no credential with image of `ansible-tower/ansible-runner`? Even if that's the case, it's not as obvious as it might seem, and it would be good to have an example.